src/riak_cs_riak_client.erl

```diff
@@ -331,6 +331,8 @@ get_user_with_pbc(MasterPbc, Key, true) ->
 get_user_with_pbc(MasterPbc, Key, false) ->
     case strong_get_user_with_pbc(MasterPbc, Key) of
         {ok, _} = OK -> OK;
+        {error, <<"{pr_val_unsatisfied,", _/binary>>} ->
```

Comment:
The PR which includes this commits have been merged already. Please create new PR for the change :smile: 

Assuming 2,000 requests and 100 bytes per log line, log size 11MB per
minutes or 670 MB per hour. If lagger setting were default, 5-minute old log
would start disappearing.